index.html

```diff
@@ -420,14 +419,22 @@
           settings</var>' <a>top-level browsing context</a>'s <a>origin</a>
         </li>
       </ol>
+      <dl class='idl' title='dictionary PermissionStorage'>
```

Comment:
That sounds good. I was going to suggest changing the name as well, but I couldn't think of a way of making it clear in the name.